src/compiler/commandLineParser.ts

```diff
@@ -2108,10 +2108,9 @@ namespace ts {
 
         function isAllowedOption({ category, name }: CommandLineOption): boolean {
             // Skip options which do not have a category or have category `Command_line_Options`
-            // Exclude all possible `Advanced_Options` in tsconfig.json which were NOT defined in command line

```

Comment:
Return the comment, remove the extra `;`

Reply:
done

Reply:
The comment is still gone. Can you please fix that so that its ready to merge